pom.xml

```diff
@@ -133,6 +133,7 @@
     <netty.version>4.1.45.Final</netty.version>
     <rocksdb.version>5.15.10</rocksdb.version>
     <hadoop.version>2.7.3</hadoop.version>
+    <hive.version>2.3.6</hive.version>
```

Comment:
can we only support a specific hive client version? how did you choose 2.3.6, which apache hadoop or cdh versions is it compatible with

Reply:
@gpang can you please comment on the hive client requirementS

Reply:
based on @gpang  hive version capatibility testing, most operations are the same in 2.2.0 and 2.3.6, but 2.3.6 has a new get_table_request(which is not used by hms tool) which does not support older versions. Although this operation is not used by hms tool so far, to be safe, will change to 2.2.0 and tested it out.